src/components/Base/Base.tsx

```diff
@@ -4,22 +4,29 @@ import styled, { css } from 'styled-components'
 type Props = {
   children: ReactNode
   radius?: 's' | 'm'
+  dialog?: boolean
   className?: string
 }
 
-export const Base = forwardRef<HTMLDivElement, Props>(({ radius = 'm', ...props }, ref) => {
-  const radiusMap = {
-    s: '6px',
-    m: '8px',
-  }
-  return <Wrapper radius={radiusMap[radius]} ref={ref} {...props} />
-})
+export const Base = forwardRef<HTMLDivElement, Props>(
```

Comment:
Sorry, The intention was not properly communicated... 😭 
I'm wondering if this style change needs to be provided on the Base component side.

I think I should extend Base on the InformationPanel component side.

```tsx
// InformationPanel.tsx
const Wrapper = styled(Base)<{ themes: Theme }>`
  ${({ themes }) => {
    const { pxToRem, space } = themes.size

    return css`
      padding: ${pxToRem(space.S)};
      box-shadow: rgba(51, 51, 51, 0.3) 0px 4px 10px 0;
    `
  }}
`
```